iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -2112,4 +2118,10 @@ public List<FileStatus> getMergeTaskInputFiles(Properties properties) throws IOE
   public MergeTaskProperties getMergeTaskProperties(Properties properties) {
     return new IcebergMergeTaskProperties(properties);
   }
+
+  @Override
+  public boolean isUndergonePartitionEvolution(org.apache.hadoop.hive.ql.metadata.Table hmsTable) {
+    Table table = IcebergTableUtil.getTable(conf, hmsTable.getTTable());
+    return table.specs().size() > 1;
```

Comment:
`HiveIcebergStorageHandler#hasUndergonePartitionEvolution` has similar logic, can we reuse?
number of specs doesn't mean that all of them are referenced

Reply:
Yes, I didn't notice hasUndergonePartitionEvolution method